recipes/org-noter

```diff
@@ -1 +1,5 @@
-(org-noter :fetcher github :repo "weirdNox/org-noter")
+(org-noter :fetcher github
+           :repo "org-noter/org-noter"
+           :branch "master"
```

Comment:
Setting the branch is only necessary if you want to use a branch other than the default branch.

Reply:
removed per request